src/GrumPHP/Task/Brunch.php

```diff
@@ -60,8 +64,10 @@ public function run(ContextInterface $context)
         }
 
         $arguments = $this->processBuilder->createArgumentsForCommand('brunch');
-        $arguments->addOptionalArgument('%s', $config['task']);
-        $arguments->addOptionalArgument('%s', $config['brunch_file']);
+        $arguments->addRequiredArgument('%s', $config['task']);
+        $arguments->addOptionalArgument('--env %s', $config['env']);
```

Comment:
This will result in the escaped CLI argument `'--env production'`, shouldn't this be 2 separate arguments like this: `'--env' 'production'`?
If so, you could use the `$arguments->addOptionalArgumentWithSeparatedValue()` method.